lib/reply.js

```diff
@@ -666,18 +667,20 @@ function sendStream (payload, res, reply) {
   } else {
     reply.log.warn('response will send, but you shouldn\'t use res.writeHead in stream mode')
   }
-  payload.pipe(res)
+
+  sendStreamTrailer(payload, res, reply)
+
+  payload.pipe(res, { end: reply[kReplyTrailers] === null })
```

Comment:
Remove the `sendStreamTrailer` function and incoroporate the if that's in there here.